construction/demo/res_config_settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- Currently not working -->
```

Comment:
This can be activated now, since https://github.com/odoo/odoo/pull/231722 is merged